src/sentry/incidents/utils/process_update_helpers.py

```diff
@@ -74,6 +75,92 @@ def get_aggregation_value_helper(subscription_update: QuerySubscriptionUpdate) -
     return aggregation_value
 
 
+def get_eap_aggregation_value(
```

Comment:
This code was added in https://github.com/getsentry/sentry/pull/81948/files#diff-d262481ba0aaae3473d14f62d3cbb554e099bda1093454b7935890139f3fe5a8 so it seems to be EAP related

Reply:
👍 yeah - seems alright to move into a method like this.

cc @shruthilayaj just incase to confirm